registry-requirements.md

```diff
@@ -13,86 +12,34 @@ From the charter:
-  - TODO: How to identify a higher-level binding in the form
-- A binding that uses a protocol MUST map at least one WoT operation (`op` keyword values such as `readproperty`) to a protocol message and vice versa
-  - Reasons Behind the Requirement:
-    - Otherwise, it is not binding the protocol to WoT and cannot be useful.
-- A binding that uses a serialization format via the `contentType` keyword, MUST mention how the Data Schema terms should be used to describe the messages.
-  - Reasons Behind the Requirement:
-    - Avoid submission of a binding like "XML Binding" that only says "Use `contentType:application/xml` and nothing more. That alone would not be enough to serialize correct messages based on the data schema.
-  - TODO: We will need additional mechanisms (including vocabulary terms) to ensure that it is possible to use other media types.
-- TODO: Explain that there are no categories
-- TODO: Analyze <https://github.com/w3c/ttwg/tree/main/boilerplate/registry>
-
-### Content of Registry Definition
+It is unrealistic to incorporate a complete list of bindings into a REC document before its publication; thus, we need a more flexible mechanism.
 
-A preliminary list of rules that is extending https://www.w3.org/2023/Process-20230612/#reg-def :
+## Content of Registry Definition
 
-- Entry format (i.e. what is put into the TD document and not what the linked document should contain)
-  - Name of the binding
-  - Link to the binding: Stable link
-  - (possibly) The binding prefix
-- Requirements on the submitted document:
-  - To be clarified but the initial list for protocols at <https://w3c.github.io/wot-binding-templates/#creating-a-new-protocol-binding-template-subspecification> and <https://w3c.github.io/wot-binding-templates/#protocol-bindings-table>
+A preliminary list of rules that is extending https://www.w3.org/2023/Process-20230612/#reg-def (needs more iteration):
 
-Aspects to clarify based on the analysis of other registries above:
+### Entry format 
 
-- Versioning of registry entries (see https://github.com/w3c/wot/tree/main/registry-analysis#versioning) and versioning with respect to 
-- Deletion and deprecation (see https://github.com/w3c/wot/tree/main/registry-analysis#deletion-and-deprecation-of-registry-entries)
-- Differentiating entry into the registry and update
-- Technical submission mechanism. How does a binding get submitted? Is it a PR, an issue linking to an existing document, or an email? See the submission mechanism fields above.
-- Whether we should have one or multiple registries
+This is what is put into the table and not what the binding should contain.
+At each submission, these information must be present in the submission.
 
+- Name of the binding
+  - Examples: `HTTP Binding Template`, `CoAP Binding Template`, 
+- Link to the binding: Stable and dated link
+  - Examples: `https://w3c.github.io/wot-binding-templates/bindings/protocols/http/index.html`
```

Comment:
This example URL should be dated.

Reply:
We don't have dated versions of them yet :/ We can put a date of last update?

Reply:
Or use an `example` URL... I'm guessing at what such a dated URL might look like --
```suggestion
  - Examples: `https://w3c.example/wot-binding-templates/bindings/protocols/http/index-20240726.html`
```

Reply:
```suggestion
  - Examples: `https://www.w3.org/TR/wot/binding-templates/http-20240726/index.html`
```

I am not sure but it should start at least like this. Given this is just an example at this point which gives both the stability and date properties, it should be fine.